examples/libuv/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017, Aaditya Kalsi
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-libuv)
+
+# download libuv
+hunter_add_package(libuv)
+
+if(NOT EXISTS "${libuv_LICENSES}")
+  message(FATAL_ERROR "File not found: ${libuv_LICENSES}")
+endif()
+
+message("License: ${libuv_LICENSES}")
+message("CMAKE_PREFIX_PATH: ${CMAKE_PREFIX_PATH}")
+
+find_package(libuv CONFIG REQUIRED)
+add_executable(example example.c)
+
+target_link_libraries(example uv)
```

Comment:
Include path still not fixed because here we should use `libuv::uv`, not `uv`. There is no target `uv` so CMake thinking that it's just system library (i.e. try to add `-luv` to the linker). The rule of thumb is to always use `foo::foo` syntax in such cases. I've updated example and hit the error `Threads::Threads target not found`. This is because `*Config.cmake` file is not calling `FindPackage(Threads)`. The fix (just like previous one) already mentioned by me in this review, please read carefully:
* https://github.com/hunter-packages/libuv/pull/1/files